app/src/main/java/org/mozilla/fenix/nimbus/view/NimbusExperiments.kt

```diff
@@ -0,0 +1,68 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.nimbus.view
+
+import android.content.res.Configuration
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.items
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.tooling.preview.Preview
+import org.mozilla.experiments.nimbus.AvailableExperiment
+import org.mozilla.fenix.compose.list.TextListItem
+import org.mozilla.fenix.theme.FirefoxTheme
+
+/**
+ * List of Nimbus Experiments.
+ *
+ * @param experiments List of [AvailableExperiment] that are going to be displayed.
+ * @param onExperimentClick Invoked when the user clicks on an [AvailableExperiment].
+ */
+@Composable
+fun NimbusExperiments(
+    availableExperiments: List<AvailableExperiment> = listOf(),
```

Comment:
My prior review changes to the `@param experiments` meant to imply that I also wanted the actual parameter name to be refactored.

```suggestion
    experiments: List<AvailableExperiment> = listOf(),
```